### PR TITLE
Typescript config

### PR DESCRIPTION
**Pull request by jjuli93**
Linter kickoff 😬

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    '@typescript-eslint/no-this-alias': 'error',
+    'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': 'error',
+    'no-use-before-define': 'off',
+    '@typescript-eslint/no-use-before-define': 'error',
+    '@typescript-eslint/no-var-requires': 'error',
+    '@typescript-eslint/prefer-namespace-keyword': 'error',
+    '@typescript-eslint/triple-slash-reference': 'error',
+    '@typescript-eslint/type-annotation-spacing': 'error',
+    'import/order': [
```
Comment (damfinkel) [inline index.js]: I'd move this one to the react eslint config

File: index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      'error', {
+        'newlines-between': 'always',
+        groups: ['builtin', 'external', 'unknown', 'parent', 'internal', 'sibling', 'index']
+      }
+    ]
+  },
+  settings: {
```
Comment (damfinkel) [inline index.js]: I wouldn't add this. This should depend on each project

File: package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  ],
+  "author": "Julián Alvarez <julian.alvarez@wolox.com.ar>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Wolox/eslint-config-wolox-typescript/issues"
+  },
+  "homepage": "https://github.com/Wolox/eslint-config-wolox-typescript#readme",
+  "peerDependencies": {
+    "eslint": "6.3.0",
+    "eslint-import-resolver-babel-module": "^5.1.0",
```
Comment (damfinkel) [inline package.json]: wouldn't have this here

**Review by damfinkel — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (jjuli93) [reply index.js]: 👌

**Review by damfinkel — APPROVED** [review]

**Pull request merged**